src/ui/SimpleFilter/SimpleFilter.ts

```diff
@@ -115,7 +117,36 @@ export class SimpleFilter extends Component {
      * <div class='CoveoSimpleFilter' data-field='@myotherfield' data-value-caption='{"txt":"Text files","html":"Web page"}'></div>
      * ```
      */
-    valueCaption: ComponentOptions.buildJsonOption()
+    valueCaption: ComponentOptions.buildJsonOption(),
+    /**
+     * See {@link IGroupByRequest.sortCriteria} for a description of each possible value.
+     *
+     * **Allowed values:**
+     *
+     * - `"score"`
+     * - `"occurrences"`
+     * - `"alphaascending"`
+     * - `"alphadescending"`
+     * - `"chisquare"`
+     *
+     */
+    availableSorts: ComponentOptions.buildListOption<'score' | 'occurrences' | 'alphaascending' | 'alphadescending' | 'chisquare'>({
```

Comment:
This is not needed for this component. 

This is used in facet to allow the end user to switch to those available sorts, but in the simple filter component this is not needed nor available for the end user.

If we want to validate the list, this can be done by sharing a module between the Facet and SimpleFilter, which would be in charge of verifying that a string is in fact a valid group by sort criteria.